presets/hugo/kool.yml

```diff
@@ -0,0 +1,3 @@
+scripts:
+  hugo: kool docker -p 1313:1313 klakegg/hugo
```

Comment:
can't we use hugo entrypoint that we already have inside the container with `kool exec`?

Reply:
TBH I didn't give much thought into this. The good thing about using `kool docker` instead of `kool exec` is not depending on the running container. So the option would be `kool exec app hugo`. I am fine either way, let me know what you think is best!